### PR TITLE
UI integration: complete documentation

### PR DESCRIPTION
**Pull request by blankoworld**
* Reorganizes UI integration page
* Adds more details about directories and files

Co-Authored-by: Olivier DOSSMANN <git@dossmann.net>

### CODE REVIEW

File: rero-instances/rero-ils-ui/ui-integration.md [hunk 1]
```diff
@@ -1,31 +1,34 @@
...
 git checkout rero/pr/xxx
 npm install
 npm run pack
 ```
 
-Then copy rero-rero-ils-ui-0.0.x.tgz into RERO-ils directory.
+Then copy **rero-rero-ils-ui-0.0.x.tgz** into RERO-ils directory.
 
-### In RERO-ILS
+`cp rero-rero-ils-ui-0.0.x.tgz path/to/rero-ils`
```
Comment (AoNoOokami) [inline rero-instances/rero-ils-ui/ui-integration.md]: Are .tgz files in gitignore?

File: rero-instances/rero-ils-ui/ui-integration.md [hunk 2]
```diff
@@ -1,31 +1,34 @@
...
 ## From a PR
 
-### In RERO-ILS-UI
+This should be done in several steps.
 
-First:
+First, in rero-ils-ui:
 
 ```bash
+cd path/to/rero-ils-UI
```
Comment (Garfield-fr) [inline rero-instances/rero-ils-ui/ui-integration.md]: `rero-ils-UI` to `rero-ils-ui`

**Review by Garfield-fr — APPROVED** [review]

File: rero-instances/rero-ils-ui/ui-integration.md [hunk 3]
```diff
@@ -1,31 +1,34 @@
...
 
-### In RERO-ILS
+`cp rero-rero-ils-ui-0.0.x.tgz path/to/rero-ils`
 
-And finally:
+Finally use **bootstrap script** with this new file:
 
 ```bash
-poetry run bootstrap -t path/to/rero-rero-ils-ui-0.0.x.tgz
+cd path/to/rero-ILS
```
Comment (iGormilhit) [inline rero-instances/rero-ils-ui/ui-integration.md]: If https://github.com/rero/developer-resources/pull/26/files#r457387476 then
```suggestion
cd path/to/rero-ils
```
:slightly_smiling_face:

File: rero-instances/rero-ils-ui/ui-integration.md (hunk 1 above)
Comment (blankoworld) [reply rero-instances/rero-ils-ui/ui-integration.md]: They are not.

**Review by iGormilhit — APPROVED** [review]

**Pull request merged**